examples/fungible/src/lib.rs

```diff
@@ -9,6 +9,8 @@ use std::collections::BTreeMap;
 /// An operation.
 #[derive(Deserialize, Serialize)]
 pub enum Operation {
+    /// Obtain the total balance of the accounts
+    TotalBalance { },
```

Comment:
Did you mean to add a query?

Reply:
Yes, I wanted to have the possibility to retrieve the total balance. This is to make the control of the accounts more stringent, that is that there was no money in other accounts. But I had some problem to make it work completely, so I should have eliminated that part.

Reply:
Well, for one it should be in the `enum Query`